### PR TITLE
6155 API change properties with the Date data type to DateTime. Repre…

### PR DESCRIPTION
**Pull request by doug-lovett**
…sent as UTC.

Signed-off-by: Doug Lovett <doug@diamante.ca>

### CODE REVIEW

**Review by kialj876 — APPROVED** [review]

File: ppr-api/src/ppr_api/models/registration.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ class RegistrationTypes(Enum):
     def json(self) -> dict:
         """Return the registration as a json object."""
         registration = {
-            'baseRegistrationNumber': self.base_registration_num,
+            'baseRegistrationNumber': self.base_reg_num,
```
Comment (thorwolpert) [inline ppr-api/src/ppr_api/models/registration.py]: i liked the verbose name ...

File: ppr-api/src/ppr_api/models/registration.py [hunk 2]
```diff
@@ -19,12 +19,12 @@
 
 from enum import Enum
 from http import HTTPStatus
-from datetime import date
 import json
 
 #from sqlalchemy import event
 
-from ppr_api.utils.datetime import format_ts, now_ts, now_ts_offset
+from ppr_api.utils.datetime import format_ts, now_ts, now_ts_offset,\
```
Comment (thorwolpert) [inline ppr-api/src/ppr_api/models/registration.py]: i should've commented earlier on this.
my thoughts when making the datetime wrapper was to make it datetime, with more explicit support for timezones, mostly UTC
note sure these other helpers should be in the module wrapper.

File: ppr-api/src/ppr_api/utils/datetime.py [hunk 3]
```diff
@@ -50,17 +49,30 @@ def now_ts_offset(offset_days: int = 1, add: bool = False):
     return now - timedelta(days=offset_days)
 
 def expiry_dt_from_years(life_years: int):
-    """Create a date representing the current date adjusted by the life_years number of years in the future."""
+    """Create a date representing the current UTC date at 23:59:59 adjusted by
```
Comment (thorwolpert) [inline ppr-api/src/ppr_api/utils/datetime.py]: see my way-late comments on this above.

**Review by thorwolpert — COMMENTED** [review]
need some discussion about date helper methods

File: ppr-api/src/ppr_api/models/registration.py (hunk 1 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/models/registration.py]: Property renamed to base_registration_num.

File: ppr-api/src/ppr_api/models/registration.py (hunk 2 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/models/registration.py]: Moved helper datetime functions to models/utils.py.

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**